bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -342,8 +342,8 @@ spec:
   - name: OpenTelemetry Operator
     url: https://github.com/open-telemetry/opentelemetry-operator
   maintainers:
-  - email: jpkroehling@redhat.com
-    name: Juraci Paixão Kröhling
+  - email: cncf-opentelemetry-contributors@lists.cncf.io
+    name: OpenTelemetry Operator maintainers
```

Comment:
I'd rather list @VineethReddy02 here as the other maintainer, as it gives people a concrete point of contact.

Reply:
and do you want to list all collector maintainers as well? 

By using specific contacts this will get outdated (your email is a great example of it). This is a community project at the end.

Reply:
>and do you want to list all collector maintainers as well?

No, just the maintainers actively working on the operator. Currently, that's me and @VineethReddy02.

Reply:
in that case I am closing this one. It's pity that CSV does not allow to set URL to this repo instead.